lang/python/pyxdg/Makefile

```diff
@@ -0,0 +1,41 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pyxdg
+PKG_VERSION:=0.26
+PKG_RELEASE:=1
+PKG_LICENSE:=LGPL-2.0
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/pyxdg
+PKG_HASH:=fe2928d3f532ed32b39c32a482b54136fe766d19936afc96c8f00645f9da1a06
+
+PKG_BUILD_DEPENDS:=python
+
+include $(INCLUDE_DIR)/package.mk
+include ../python-package.mk
+
+define Package/pyxdg
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=pyxdg
+  URL:=http://freedesktop.org/wiki/Software/pyxdg
+  DEPENDS:=+python
+endef
+
+define Package/pyxdg/description
+  A Python module to deal with freedesktop.org specifications
+endef
+
+define Build/Compile
```

Comment:
you can remove `Build/Compile` & `Package/pyxdg/install` and add a `VARIANT:=python` tag
```
define Package/python-pyxdg
  SUBMENU:=Python
  SECTION:=lang
  CATEGORY:=Languages
  TITLE:=pyxdg
  URL:=http://freedesktop.org/wiki/Software/pyxdg
  DEPENDS:=+python
  VARIANT:=python
endef
```

and call
```
$(eval $(call PyPackage,python-pyxdg))
$(eval $(call BuildPackage,python-pyxdg))
```
this should be more compact;

it's good to also rename to `python-pyxdg` in case there will be a `python3-pyxdg` variant later

Reply:
I forgot to add about `PKG_BUILD_DIR` & `PKG_UNPACK`

there is a readme for python pacakges 
https://github.com/openwrt/packages/blob/master/lang/python/README.md

Reply:
```
.../staging_dir/hostpkg/bin/python2.7   ./setup.py  install --prefix="/usr" --root="/mnt/misc/develop/mir3g/git/my/openwrt/build_dir/target-mipsel_24kc_musl/python-pyxdg-0.26/ipkg-install" --single-version-externally-managed ;
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --single-version-externally-managed not recognized

```

how to fix ?

Reply:
i'll need to document that

add
```
PYTHON_PKG_SETUP_ARGS:=
PYTHON3_PKG_SETUP_ARGS:=
```

after `include python[3]-package.mk`